src/shadowbox/server/manager_service.ts

```diff
@@ -52,12 +52,12 @@ interface RequestParams {
   id?: string;
   name?: string;
   metricsEnabled?: boolean;
-  limit?: DataUsage|{};
+  limit?: DataUsage;
```

Comment:
I think the best solution here is to make `RequestParams` be the type `[key: string]: unknown`.  This accurately reflects what we know about `params` when we receive it.

The functions would then look like
```ts
const id = params['id']
if (typeof id !== "string") { return ... }
// id is now a string
```